Dockerfile

```diff
@@ -31,13 +31,14 @@ RUN apt-get update \
        unzip \
     && rm -rf /var/lib/apt/lists/*
 
-# Install Google Cloud SDK and Gcloud Auth Plugin
-RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
-    && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
-
-RUN apt-get update \
-    && apt-get install -y google-cloud-sdk google-cloud-sdk-gke-gcloud-auth-plugin \
-    && rm -rf /var/lib/apt/lists/*
+# Install Google cli so kubectl works w/ gke clusters
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
```

Comment:
I believe that /tmp/google-cloud-sdk.tar.gz is downloaded but never deleted. please delete after download to reduce image size.